content/en/blog/2019/app-identity-and-access-adapter/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: App Identity and Access Adapter
+subtitle: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes
+description: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes.
+publishdate: 2019-09-01
+attribution: Anton Aleksandrov (IBM)
+keywords: [security,oidc,jwt,policies]
+---
+
+Whether your computing environment uses multiple cloud providers, a single cluster, a combination of on- and off-premise solutions, or multiple containers in one cloud, having a centralized identity management can help you to preserve existing infrastructure and avoid vendor lock-in.
+
+Simply put, if you are running your containerized applications on Kubernetes, you can benefit from using the App Identity and Access Adapter for an abstracted level of security with zero code changes and no redeployments.
```

Comment:
I think you could remove "Simply put," and make this sentence the beginning sentence of your post. It's great because it gets to the heart of the matter immediately/concisely, and draws the reader in quicker.